tests/testColor.py

```diff
@@ -183,15 +165,13 @@ def testCtorFromMetadata(self):
 
         self.assertAlmostEqual(self.calib.getMagnitude(flux, fluxErr)[1], 2.5/math.log(10)*fluxErr/flux)
 
-        #
         # Check that we can clean up metadata
-        #
         afwImage.stripCalibKeywords(metadata)
         self.assertEqual(len(metadata.names()), 0)
 
     def testCalibEquality(self):
         self.assertEqual(self.calib, self.calib)
-        self.assertFalse(self.calib != self.calib)
+        self.assertFalse(self.calib != self.calib)  # using assertFalse to directly test =! operator
```

Comment:
typo in comment `=!` but I like the comment there to stop people trying to delete the line.